src/modules/course/course.service.ts

```diff
@@ -63,9 +64,10 @@ export class CourseService {
   }
 
   // fetch brief info for cards for all courses
-  async getBreifAllCourses(): Promise<Course[]> {
+  async getBreifAllCourses(skipNum: string): Promise<Course[]> {
```

Comment:
why you are taking it as a string?

Reply:
query is of type string, so it can be parsed to integer